calico_node/Makefile

```diff
@@ -469,6 +472,64 @@ release: clean
 	@echo "docker push quay.io/$(NODE_CONTAINER_NAME):latest"
 	@echo "See RELEASING.md for detailed instructions."
 
+
+RELEASE_DIR:=release-$(CALICO_VER)
+RELEASE_DIR_K8S_MANIFESTS:=$(RELEASE_DIR)/k8s-manifests
+MANIFEST_SRC := ../_site/$(RELEASE_STREAM)/getting-started/kubernetes/installation
+
+## Create an archive that contains a complete "Calico" release
+release-image: $(RELEASE_DIR).tgz
+
+$(RELEASE_DIR).tgz: $(RELEASE_DIR) $(RELEASE_DIR_K8S_MANIFESTS) $(RELEASE_DIR)/calico-node.tar $(RELEASE_DIR)/calico-typha.tar $(RELEASE_DIR)/calico-cni.tar $(RELEASE_DIR)/calico-kube-policy-controller.tar $(RELEASE_DIR)/calicoctl $(RELEASE_DIR)/README
+	tar -czvf $(RELEASE_DIR).tgz $(RELEASE_DIR)/*
+
+$(RELEASE_DIR)/README:
+	@echo "This directory contains a complete release of Calico $(CALICO_VER)" >> $@
+	@echo "" >> $@
+	@echo "It contains:" >> $@
+	@echo "* The calico/node docker image  (version $(CALICO_VER))" >> $@
+	@echo "* The calico/typha docker image  (version $(TYPHA_VER))" >> $@
+	@echo "* The calico/cni docker image  (version $(CNI_VER))" >> $@
+	@echo "* The calico/kube-policy-controller docker image (version $(K8S_POLICY_VER))" >> $@
+	@echo "* Kubernetes manifests in the k8s-manifests directory" >> $@
+	@echo "* The calicoctl binary (version $(CALICOCTL_VER))" >> $@
```

Comment:
I think it might be worth including a link to the correct version of the docs (although only worth doing if we aren't going to go and change our docs URL in the future).